### ISSUE TITLE
DSpellCheck.ini cannot be written

### ISSUE THREAD

**Issue by mathaifenn** [Predelnik/DSpellCheck#138]
I recently installed notepad++ from the software centre on ubuntu 18.04. but when trying to change the configuration of DSpellCheck I get a message saying 

Setting file z:\home\mathai\snap\notepad-plus-plus\42\notepad-plus-plus\plugins\Config\DSpellCheck.ini cannot be written. All settings will be lost when you close Notepad++

**Comment by Predelnik** [comment]
Is N++ run through wine in this case or how does it work?
I will try to test this but probably not super soon.

**Comment by mathaifenn** [comment]
Yes through wine. I was surprised that it was available in the software
centre
